tests/e2e/src/pages/main.page.ts

```diff
@@ -29,6 +29,10 @@ export class MainPage extends BasePage {
     return "//*[@class='modal-card']";
   }
 
+  get circleTimerElement() {
+    return "//*[@class='radial-progress ml-1.5 cursor-help']";
```

Comment:
We cannot use dynamic (with spaces) classes as a xpath string. If you are sure about some specific class you can take it by contains(class, value)

Reply:
Fixed in latest commit